cart-service/.gitignore

```diff
@@ -0,0 +1,4 @@
+/.idea/
```

Comment:
This .gitignore file does not cover all the ignore files. You can use it from other services. ex. order-service

Reply:
Done